.github/workflows/scheduled.yml

```diff
@@ -0,0 +1,128 @@
+name: Scheduled
+
+on:
+  pull_request:
+    paths:
+      - scripts/get_scarb_versions.sh
+      - .github/workflows/scheduled.yml
+  schedule:
+    - cron: '0 0 * * 3,6'
```

Comment:
Nit, this could maybe run on Wednesday and Sunday instead? If it fails on a weekend, it will have to wait until Monday to be resolved anyway.